rosbag2_transport/test/rosbag2_transport/record_integration_fixture.hpp

```diff
@@ -50,21 +47,33 @@ class RecordIntegrationTestFixture : public Rosbag2TransportTestFixture
 
   void TearDown() override
   {
+    stop_spinning();
```

Comment:
@clalancette To avoid potential freezes in tests, I think it makes sense to call `stop_spiining()` after the `rclcpp::shutdown()` in this destructor.